db/db_test.cc

```diff
@@ -2587,15 +2585,23 @@ struct MTThread {
   bool multiget_batched;
 };
 
+// Based on Peter Dillinger's suggestion, the threads have been changed from
```

Comment:
Not necessary. Thanks. Someone can use `git blame` if they want to see the history of this test.

Reply:
Done